kolibri/plugins/management/assets/src/vue/top-nav/index.vue

```diff
@@ -35,6 +39,14 @@
       },
     },
     computed: {
+      classesLink() {
+        return { name: constants.PageNames.CLASS_MGMT_PAGE };
+      },
+      classesActive() {
+        return this.pageName === constants.PageNames.CLASS_MGMT_PAGE ||
+          this.pageName === constants.PageNames.CLASS_EDIT_MGMT_PAGE ||
+          this.pageName === constants.PageNames.CLASS_ENROLL_MGMT_PAGE;
+      },
```

Comment:
this could also be written:

```javascript
classesActive() {
  return [
    constants.PageNames.CLASS_MGMT_PAGE,
    constants.PageNames.CLASS_EDIT_MGMT_PAGE,
    constants.PageNames.CLASS_ENROLL_MGMT_PAGE,
  ].includes(this.pageName);
},
```